packages/next/build/webpack-config.ts

```diff
@@ -1272,6 +1273,11 @@ export default async function getBaseWebpackConfig(
       ].filter(Boolean),
     },
     plugins: [
+      ...(!isServer &&
```

Comment:
This still applies in development where eval sourcemaps are already enabled. Is that intended?